packages/notionable/src/stores/root-store.ts

```diff
@@ -0,0 +1,82 @@
+
+import { NetworkClientModel } from '@notional-finance/core-entities';
+import { types, Instance, } from 'mobx-state-tree';
+import { PortfolioStoreModel } from './portfolio-store';
+import { getFromLocalStorage, Network, THEME_VARIANTS } from '@notional-finance/util';
+import { createContext } from 'react';
+import { AppStoreModel } from './app-store';
+
+
+export type RootStoreType = Instance<typeof RootStore>
+export type NetworkClientModelType = Instance<typeof NetworkClientModel>
+export type AppStoreModelType = Instance<typeof AppStoreModel>
+export type PortfolioStoreModelType = Instance<typeof PortfolioStoreModel>
+
+const userSettings = getFromLocalStorage('userSettings');
+
+const RootStore = types.model("RootStore", {
+  mainnetStore: NetworkClientModel, 
+  arbitrumStore: NetworkClientModel, 
+  allNetworksStore: NetworkClientModel, 
+  portfolioStore: PortfolioStoreModel,
+  appStore: AppStoreModel,
+}).actions(self => {
+  const getNetworkClient = (network: Network) => {
+    switch (network) {
+      case Network.mainnet:
+        return self.mainnetStore
+      case Network.arbitrum:
+        return self.arbitrumStore
+      default:
```

Comment:
All networks is not default, is is Network.all, throw an error on default like the other function